files/run_worker.sh

```diff
@@ -1,14 +1,24 @@
 #!/usr/bin/bash
 
+set -x
+
 # $APP defines where's the module (or package)
 if [[ -z ${APP} ]]; then
     echo "APP not defined or empty, exiting"
     exit 1
 fi
 
-printf "packit:x:$(id -u):0:Packit Service:/home/packit:/bin/bash\n" >>/home/packit/passwd
-
 export PACKIT_HOME=/home/packit
+
+cp ${PACKIT_HOME}/passwd.user ${PACKIT_HOME}/passwd
```

Comment:
passwd.user or passwd.packit ?

Reply:
I am such idiot :( I have forgotten to add the latest changes.
Good catch. Again;)